### PR TITLE
add zuul jobs

### PR DESCRIPTION
**Pull request by fredericlepied**
Depends-On: https://github.com/wazo-platform/sf-jobs/pull/15

### CODE REVIEW

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

File: tox.ini [hunk 1]
```diff
@@ -35,3 +34,42 @@ deps =
...
+    sh
+
+[flake8]
+# E501: line too long (80 chars)
+# W503: line break before binary operator
+exclude = .tox,.eggs,alembic
+show-source = true
+ignore = E501, W503
+max-line-length = 99
+application-import-names = wazo_webhookd
```
Comment (sduthil) [inline tox.ini]:
```suggestion
application-import-names = wazo_calld
```

File: integration_tests/suite/test_applications.py [hunk 2]
```diff
@@ -1163,6 +1163,9 @@ def test_delete(self):
             'both',
         ).json()
 
+        print(snoop_1)
+        print(snoop_2)
+
```
[the remark this answers is not in the document]
Comment (fredericlepied) [reply integration_tests/suite/test_applications.py]: No idea

[the remark this answers is not in the document]
Comment (sduthil) [reply integration_tests/suite/test_applications.py]: I just confirmed with @pc-m: they're not :) I'll remove them

**Comment by sileht** [comment]
recheck

**Comment by fredericlepied** [comment]
regate

**Comment by fredericlepied** [comment]
regate

File: tox.ini [hunk 3]
```diff
@@ -35,3 +34,43 @@ deps =
...
+
+[testenv:linters]
+skip_install = true
+basepython = python3
+deps = flake8
+       flake8-colors
+       black
+commands =
+  # Don't fail on black failure for now
+  -black --skip-string-normalization --check .
```
Comment (sileht) [inline tox.ini]: Maybe you can add a commit with the code blackified and remove the '-'. Otherwise I'm good.

Comment (fredericlepied) [reply tox.ini]: That's a long standing one that is failing in the integration tests not really a linter issue...

**Comment by pc-m** [comment]
I removed the `mergeit` because I added some debug prints that I would not want merged when the job start working

**Comment by pc-m** [comment]
recheck

**Comment by pc-m** [comment]
recheck

File: tox.ini (hunk 3 above)
Comment (fblackburn1) [reply tox.ini]: Please don't blackify the code before merging all branches (particularly `use-real-calld-client`, still need ready)

**Comment by pc-m** [comment]
Depends-On: https://github.com/wazo-platform/wazo-calld/pull/54

**Comment by pc-m** [comment]
recheck

**Comment by pc-m** [comment]
> Depends-On: #54

I'll wait for this to get reviewed before adding `mergeit` to avoid pulling in the bug fix without someone accepting it

**Comment by pc-m** [comment]
recheck

**Pull request merged**